cpp/src/community/k_truss_impl.cuh

```diff
@@ -121,6 +314,8 @@ k_truss(raft::handle_t const& handle,
     edge_weight{std::nullopt};
```

Comment:
Should we maintain `edge_weight` from here? We can treat the graph as an unweighted graph till we finish finding K-Truss edge sources & destinations. Once we have K-Truss edge sources & destinations, we can extract edge (src, dst, weight) triplets from the input graph.